### PR TITLE
Coding: Develop First Time Issue Section of Contribute Page 

### LINKED ISSUE #67: Coding: Develop First Time Issue Section of Contribute Page
## Description
As a developer,
I need the following section of mockup to be developed,
so that I can add as First Time Issue Component [here](https://github.com/anitab-org/anitab-org.github.io/tree/develop/src/Components).

## Mocks
code-first-time-issues

Font Size for heading SOME COOL FIRST TIME ISSUES TO WORK ON: Header Medium(32pt) weight- Medium
Font Size for the description below: body 18pt weight- Light
Font Style- Avenir
Font Color (heading): #103B81
Font Color (Body): #103B81

[Design Link For Development
](https://xd.adobe.com/view/b3ce723d-5bc6-45f5-7035-0a637d919751-c5ff/)

### Update [Required]
- [ ] Follows all the design guidelines as mentioned and is responsive enough on large screen devices.
- [ ] Fetch First Time only issue from GitHub.
- [ ] See all issues on GitHub is a link that should redirect the user to issues page showing all first-time issues available on GitHub.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
3-5 hours

### PR DESCRIPTION
**Pull request by rutvi18**
### Description

Include a summary of the change and relevant motivation/context. List any dependencies that are required for this change.

Fixes #67 

### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
If you are working on a design then add a gif to show the responsiveness of your design.
Please follow [testing guidelines](https://github.com/anitab-org/anitab-org.github.io/wiki/Prepare-testing-GitHub-Pages-for-your-PR) and share the link of deployed site here.

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [x] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by rutvi18** [comment]
Please do see the index.js of contribute. I have implemented the code and documentation basic structure. But, I am not able to figure out how to make first paragraph

**Comment by Bucky25** [comment]
@rutvi18 thanks for the pr :) your code seems to be good please add a link to deployed gh-page repo

File: src/content/contributeIntro.json [hunk 1]
```diff
@@ -1,24 +1,62 @@
...
+             {
+               "par":"bdjbc"
+             }
+          ]
+      },
+      {
+        "title":"Code",
+        "content":[
+          {
+            "par":"hegjfgwj"
```
Comment (annabauza) [inline src/content/contributeIntro.json:35]: this random content will not be merged

**Review by annabauza — CHANGES_REQUESTED** [review]
No random text will be merged. The code looks good though. Please remove paragraphs and rise issue for the content, so we could merge this PR. Thanks.

**Review by isabelcosta — COMMENTED** [review]
@rutvi18 can you put a GIF or an image of your UI changes reflected in the project. This would make your pull request better in terms of giving context to code reviewers ⭐ 

Also, can you add some content to the pull request description, please? I see it's still the same text from the pull request template.

**Comment by tichnas** [comment]
@isabelcosta @rutvi18 It looks like this PR is incomplete and a part of #160. Should we close it?

File: src/content/contributeIntro.json (hunk 1 above)
Comment (rutvi18) [reply src/content/contributeIntro.json:35]: Ok....but what content should I write?

**Comment by rutvi18** [comment]
@tichnas few changes are left. I will update in 2-3 days. And I have closed the PR #160 as this one is the updated one.

File: src/content/contributeIntro.json (hunk 1 above)
Comment (tichnas) [reply src/content/contributeIntro.json:35]: No, you don't have to write anything yourself. Just wait till the actual content gets merged.

**Comment by tichnas** [comment]
@rutvi18 Ohk, no issues. And thanks for closing that PR :)

File: src/content/contributeIntro.json (hunk 1 above)
Comment (rutvi18) [reply src/content/contributeIntro.json:35]: Got it..thanks :)

**Comment by rutvi18** [comment]
Can anybody please tell if I can assign an issue to someone or anyone from the team need to do this?

**Comment by rutvi18** [comment]
I have merged the file from an issue #60. Please check if any more changes are required.

**Review by tichnas — CHANGES_REQUESTED** [review]
Please check once if all the changed files are added because the code doesn't seem to give the desired result.
Also, deployed link or a screenshot/GIF will be of great help to review.

File: src/content/contribute_ways.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
+{
```
Comment (annabauza) [inline src/content/contribute_ways.json:1]: where do you use this file?

**Comment by annabauza** [comment]
@tichnas I close this PR due to lack of response.

**Pull request closed without merging**